hack/cosign-sign-blob-fulcio.sh

```diff
@@ -22,94 +22,68 @@ set -o errexit
 set -o pipefail
 set -o nounset
 set -o errtrace
+# Enable debugging
+#set -o xtrace
 
 if (( $# != 1 )); then
     echo "usage $0 <file to sign>"
     exit 1
 fi
 
+# We call `cosign initialize` below which will modify the local
+# $HOME/.sigstore/root, so we keep a backup of it and restore it afterwards
+SIGSTORE_ROOT="${HOME}"/.sigstore/root
```

Comment:
FYI, the environment variable `TUF_ROOT` can be used to instrument cosign to use a different location other than `~/.sigstore/root`. Something like `export TUF_ROOT="$(mktemp -d)"` wouldn't require a backup/restore strategy.

Reply:
I thought I tried that, let me doublecheck

Reply:
Yeah, missed to `export`... works like a charm with `TUF_ROOT`